src/spec-node/containerFeatures.ts

```diff
@@ -113,6 +113,7 @@ export async function extendImage(params: DockerResolverParameters, config: Subs
 		'--target', featureBuildInfo.overrideTarget,
 		'-f', dockerfilePath,
 		...additionalImageNames.length > 0 ? additionalImageNames.map(name => ['-t', name]).flat() : ['-t', updatedImageName],
+		...imageLabels.length > 0 ? imageLabels.map(label => ['--label', label]).flat() : ['--label', imageLabels.toString()],
```

Comment:
If there are no labels, why don we want to generates a single `--label` argument with an empty value?

Reply:
changed to empty if --label is not there